index.html

```diff
@@ -775,5 +775,104 @@ <h3 id="placement-of-combination-of-group-ruby-and-group-ruby">Placement of the
 described in [[[#placement-of-group-ruby]]].</p>
 </section>
 </section>
+
+
+
+<section class="appendix">
+<h2 id="simplification-of-ruby-placement-rules">Simplification of Ruby placement rules</h2>
+
+<section>
+<h3 id="alignment-between-ruby-annotation-and-base-text">Alignment between ruby annotation and base text</h3>
+
+<section>
+<h4 id="alignment-between-ruby-annotation-and-base-text-mono-ruby">Mono-ruby</h4>
+<p>In <a title="mono-ruby">mono-ruby</a>, 
```

Comment:
Just a reminder, we should merge https://github.com/w3c/simple-ruby/pull/63 first, and then merge this, otherwise, the link here will be invalid.

Reply:
Yes, of course. (forgot to include in the description... - added)